easybuild/easyconfigs/g/gradunwarp/gradunwarp-1.1.0-foss-2019a-HCP-Python-2.7.15.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = "gradunwarp"
+version = "1.1.0"
+versionsuffix = "-HCP-Python-%(pyver)s"
+
+homepage = "https://github.com/Washington-University/gradunwarp"
+description = """Gradient Unwarping. This is the Human Connectome Project fork of the no longer maintained original."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = ['https://github.com/Washington-University/gradunwarp/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['4803b8055dbeedb0435246a525aa69f1f3425c55d57c973c045deb39bc95c955']
+
+download_dep_fail = True
+use_pip = False
```

Comment:
@crubb please add `sanity_pip_check = True` here?

Reply:
Hey, we're not using pip to install, therefore I suppose `sanity_pip_check` should be the default, i.e. `False`? 

See https://github.com/easybuilders/easybuild-easyconfigs/issues/9563

Reply:
ah, you're right, I missed that use_pip = False

Reply:
@crubb Can you clarify why you're using `use_pip = False` here, is there a specific reason for it?

Reply:
As discussed on Slack: https://github.com/easybuilders/easybuild-easyconfigs/pull/9637